Integration test failure with Icat 5

There is an integration test failure when running the tests against Icat 5:

> Running org.icatproject.ids.integration.one.FileCheckerTest
>  
> Setting up one.properties took 0.052seconds
> Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 13.583 sec <<< FAILURE! - in org.icatproject.ids.integration.one.FileCheckerTest
> everythingTest(org.icatproject.ids.integration.one.FileCheckerTest)  Time elapsed: 13.018 sec  <<< FAILURE!
> java.lang.AssertionError: Datafile 30890 (uploaded_file_2) file size wrong:file size null
> 	at org.junit.Assert.fail(Assert.java:89)
> 	at org.junit.Assert.assertTrue(Assert.java:42)
> 	at org.icatproject.ids.integration.one.FileCheckerTest.checkHas(FileCheckerTest.java:98)
> 	at org.icatproject.ids.integration.one.FileCheckerTest.everythingTest(FileCheckerTest.java:65)

> Running org.icatproject.ids.integration.two.FileCheckerTest
> 
> Setting up two.properties took 0.083seconds
> About to wait
> /home/glassfish/test/data/ids/archive/490/1471 will be truncated to 300
> Watching /home/glassfish/test/data/ids/errorLog
> /home/glassfish/test/data/ids/archive/490/1471 will be truncated to 0
> Watching /home/glassfish/test/data/ids/errorLog
> Watching /home/glassfish/test/data/ids/errorLog
> Watching /home/glassfish/test/data/ids/errorLog
> Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 70.887 sec <<< FAILURE! - in org.icatproject.ids.integration.two.FileCheckerTest
> everythingTest(org.icatproject.ids.integration.two.FileCheckerTest)  Time elapsed: 21.259 sec  <<< FAILURE!
> java.lang.AssertionError: Dataset 1484 (ds1_1662980655755) datafile uploaded_file_2 file size wrong:file size null
> 	at org.junit.Assert.fail(Assert.java:89)
> 	at org.junit.Assert.assertTrue(Assert.java:42)
> 	at org.icatproject.ids.integration.two.FileCheckerTest.checkHas(FileCheckerTest.java:205)
> 	at org.icatproject.ids.integration.two.FileCheckerTest.everythingTest(FileCheckerTest.java:75)

> Results :
> 
> Failed tests: 
>   FileCheckerTest.everythingTest:65->checkHas:98 Datafile 30890 (uploaded_file_2) file size wrong:file size null
>   FileCheckerTest.everythingTest:75->checkHas:205 Dataset 1484 (ds1_1662980655755) datafile uploaded_file_2 file size wrong:file size null

This is because the default for the `fileSize` attribute is 0 in Icat 5: https://github.com/icatproject/icat.server/commit/5c6c19d3b0f8cae047d286c8ba1933f581164865, so when the test attempts to set `fileSize` to null, it is set to 0 instead.

--- a/src/test/java/org/icatproject/ids/integration/one/FileCheckerTest.java
+++ b/src/test/java/org/icatproject/ids/integration/one/FileCheckerTest.java
@@ -56,13 +56,7 @@ public class FileCheckerTest extends BaseTest {
 
 		df.setFileSize(fileSize + 1);
 		icatWS.update(sessionId, df);
-
 		checkHas("Datafile", dfid, "file size wrong");
-
-		df.setFileSize(null);
-		icatWS.update(sessionId, df);
-		Files.deleteIfExists(errorLog);
-		checkHas("Datafile", dfid, "file size null");
 
 		df.setFileSize(fileSize);
 		df.setChecksum("Aardvark");

--- a/src/test/java/org/icatproject/ids/integration/two/FileCheckerTest.java
+++ b/src/test/java/org/icatproject/ids/integration/two/FileCheckerTest.java
@@ -66,13 +66,7 @@ public class FileCheckerTest extends BaseTest {
 
 		df.setFileSize(fileSize + 1);
 		icatWS.update(sessionId, df);
-
 		checkHas("Dataset", datasetIds.get(0), "file size wrong");
-
-		df.setFileSize(null);
-		icatWS.update(sessionId, df);
-		Files.deleteIfExists(errorLog);
-		checkHas("Dataset", datasetIds.get(0), "file size null");
 
 		df.setFileSize(fileSize);
 		df.setChecksum("Aardvark");

